frontend/src/components/ButtonBig.js

```diff
@@ -0,0 +1,14 @@
+import styled from 'styled-components/macro';
+
+export default function ButtonBig() {
+    return <Button> LET'S PLAY </Button>;
+}
+
+const Button = styled.button`
+    border-radius: 50px;
```

Comment:
this is a huge border radius, for completely round corners use `border-radius: 50%`

Reply:
check ^^